ui/src/app/components/object-detail/object-detail.component.html

```diff
@@ -750,13 +750,12 @@ <h3 class="title">
       <ng-container matColumnDef="spIndexOrder">
         <th class="table_header" mat-header-cell *matHeaderCellDef>Column Order</th>
         <td mat-cell *matCellDef="let element">
-          <div *ngIf="isIndexEditMode && element.get('spOrder').value">
+          <div *ngIf="isIndexEditMode && element.get('spColName').value">
             <input
               matInput
               class="order_input"
-              type="text"
+              type="number"
```

Comment:
While adding a column back into primary key it takes the older order and results in issue : https://drive.google.com/file/d/1m5Rj2VVCWE0pCCsnTqq8SDITafzWcuRQ/view?usp=sharing&resourcekey=0-xCKQPLqlQ8SeZo81oTkzEQ . And there is no way to add that column at the end.

Reply:
Solved